rest/route/test.go

```diff
@@ -217,3 +217,56 @@ func (tgh *testGetHandler) addDataToResponse(resp gimlet.Responder, testResult i
 
 	return nil
 }
+
+///////////////////////////////////////////////////////////////////////////////
+//
+// GET  /tasks/{task_id}/tests/count
+
+type testCountGetHandler struct {
+	taskID    string
+	execution int
+}
+
+func makeFetchTestCountForTask() gimlet.RouteHandler {
+	return &testCountGetHandler{}
+}
+
+func (h *testCountGetHandler) Factory() gimlet.RouteHandler {
+	return &testCountGetHandler{}
+}
+
+func (h *testCountGetHandler) Parse(ctx context.Context, r *http.Request) error {
+	projCtx := MustHaveProjectContext(ctx)
+	if projCtx.Task == nil {
+		return gimlet.ErrorResponse{
+			Message:    "task not found",
+			StatusCode: http.StatusNotFound,
+		}
+	}
+	h.taskID = projCtx.Task.Id
+
+	var err error
+	vals := r.URL.Query()
+	execution := vals.Get("execution")
+
+	if execution != "" {
+		h.execution, err = strconv.Atoi(execution)
+		if err != nil {
+			return gimlet.ErrorResponse{
```

Comment:
`Parse` already returns `http.StatusBadRequest` by default so there's no need to pass back a `gimlet.ErrorResponse`:
```go
return errors.Wrap(err, "invalid execution number")
```

Reply:
Done.